[22/Feb/2015 22:08:33] DEBUG [django.db.backends:91] (0.001) SET SQL_AUTO_IS_NULL = 0; args=None
[22/Feb/2015 22:08:33] DEBUG [django.db.backends:91] (0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1k56cc3sgsiyiyr12qevc00dzgkm9x94' AND `django_session`.`expire_date` > '2015-02-22 22:08:33') LIMIT 21; args=('1k56cc3sgsiyiyr12qevc00dzgkm9x94', u'2015-02-22 22:08:33')
[22/Feb/2015 22:08:33] INFO [app.views:20] Create project with name Barcelona
[22/Feb/2015 22:08:33] DEBUG [django.db.backends:91] (0.021) UPDATE `django_session` SET `session_data` = 'OTQ0MWQ4ZjZmZWVkMTYxZDRiZjVjYjM1ZDEyZjMwMzU0OWQ4YzkzMjp7ImRvbWFpbiI6IkJhcmNlbG9uYSJ9', `expire_date` = '2015-03-08 22:08:33' WHERE `django_session`.`session_key` = '1k56cc3sgsiyiyr12qevc00dzgkm9x94'; args=(u'OTQ0MWQ4ZjZmZWVkMTYxZDRiZjVjYjM1ZDEyZjMwMzU0OWQ4YzkzMjp7ImRvbWFpbiI6IkJhcmNlbG9uYSJ9', u'2015-03-08 22:08:33', '1k56cc3sgsiyiyr12qevc00dzgkm9x94')
[22/Feb/2015 22:08:36] DEBUG [django.db.backends:91] (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
[22/Feb/2015 22:08:36] DEBUG [django.db.backends:91] (0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1k56cc3sgsiyiyr12qevc00dzgkm9x94' AND `django_session`.`expire_date` > '2015-02-22 22:08:36') LIMIT 21; args=('1k56cc3sgsiyiyr12qevc00dzgkm9x94', u'2015-02-22 22:08:36')
[22/Feb/2015 22:08:36] INFO [app.views:33]  Configured project with domain Barcelona and keyword malaga
[22/Feb/2015 22:08:36] DEBUG [django.db.backends:91] (0.001) SET SQL_AUTO_IS_NULL = 0; args=None
[22/Feb/2015 22:08:36] DEBUG [django.db.backends:91] (0.001) SELECT `celery_taskmeta`.`id`, `celery_taskmeta`.`task_id`, `celery_taskmeta`.`status`, `celery_taskmeta`.`result`, `celery_taskmeta`.`date_done`, `celery_taskmeta`.`traceback`, `celery_taskmeta`.`hidden`, `celery_taskmeta`.`meta` FROM `celery_taskmeta` WHERE `celery_taskmeta`.`task_id` = '040dabab-be10-46cd-b398-ba693ac28d00' LIMIT 21; args=('040dabab-be10-46cd-b398-ba693ac28d00',)
[22/Feb/2015 22:08:36] DEBUG [django.db.backends:91] (0.011) INSERT INTO `celery_taskmeta` (`task_id`, `status`, `result`, `date_done`, `traceback`, `hidden`, `meta`) VALUES ('040dabab-be10-46cd-b398-ba693ac28d00', 'FAILURE', 'gAJjZXhjZXB0aW9ucwpUeXBlRXJyb3IKcQFVMnN0YXJ0X2RpZ2dlcigpIHRha2VzIGV4YWN0bHkgMyBhcmd1bWVudHMgKDIgZ2l2ZW4pcQKFcQNScQQu', '2015-02-22 22:08:36', 'Traceback (most recent call last):\n  File \"C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py\", line 240, in trace_task\n    R = retval = fun(*args, **kwargs)\n  File \"C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py\", line 438, in __protected_call__\n    return self.run(*args, **kwargs)\nTypeError: start_digger() takes exactly 3 arguments (2 given)\n', 0, 'eJxrYKotZIzgYGBgSM7IzEkpSs0rZIotZC7WAwBWuwcA'); args=['040dabab-be10-46cd-b398-ba693ac28d00', 'FAILURE', u'gAJjZXhjZXB0aW9ucwpUeXBlRXJyb3IKcQFVMnN0YXJ0X2RpZ2dlcigpIHRha2VzIGV4YWN0bHkgMyBhcmd1bWVudHMgKDIgZ2l2ZW4pcQKFcQNScQQu', u'2015-02-22 22:08:36', 'Traceback (most recent call last):\n  File "C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py", line 240, in trace_task\n    R = retval = fun(*args, **kwargs)\n  File "C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py", line 438, in __protected_call__\n    return self.run(*args, **kwargs)\nTypeError: start_digger() takes exactly 3 arguments (2 given)\n', False, u'eJxrYKotZIzgYGBgSM7IzEkpSs0rZIotZC7WAwBWuwcA']
[22/Feb/2015 22:11:30] DEBUG [django.db.backends:91] (0.003) SET SQL_AUTO_IS_NULL = 0; args=None
[22/Feb/2015 22:11:30] DEBUG [django.db.backends:91] (0.001) SHOW TABLES; args=None
[22/Feb/2015 22:11:30] DEBUG [django.db.backends:91] (0.008) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
[22/Feb/2015 22:11:51] DEBUG [django.db.backends:91] (0.001) SET SQL_AUTO_IS_NULL = 0; args=None
[22/Feb/2015 22:11:51] DEBUG [django.db.backends:91] (0.001) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1k56cc3sgsiyiyr12qevc00dzgkm9x94' AND `django_session`.`expire_date` > '2015-02-22 22:11:51') LIMIT 21; args=('1k56cc3sgsiyiyr12qevc00dzgkm9x94', u'2015-02-22 22:11:51')
[22/Feb/2015 22:11:51] INFO [app.views:20] Create project with name Barcelona
[22/Feb/2015 22:11:51] DEBUG [django.db.backends:91] (0.057) UPDATE `django_session` SET `session_data` = 'OTQ0MWQ4ZjZmZWVkMTYxZDRiZjVjYjM1ZDEyZjMwMzU0OWQ4YzkzMjp7ImRvbWFpbiI6IkJhcmNlbG9uYSJ9', `expire_date` = '2015-03-08 22:11:51' WHERE `django_session`.`session_key` = '1k56cc3sgsiyiyr12qevc00dzgkm9x94'; args=(u'OTQ0MWQ4ZjZmZWVkMTYxZDRiZjVjYjM1ZDEyZjMwMzU0OWQ4YzkzMjp7ImRvbWFpbiI6IkJhcmNlbG9uYSJ9', u'2015-03-08 22:11:51', '1k56cc3sgsiyiyr12qevc00dzgkm9x94')
[22/Feb/2015 22:11:54] DEBUG [django.db.backends:91] (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
[22/Feb/2015 22:11:54] DEBUG [django.db.backends:91] (0.000) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1k56cc3sgsiyiyr12qevc00dzgkm9x94' AND `django_session`.`expire_date` > '2015-02-22 22:11:54') LIMIT 21; args=('1k56cc3sgsiyiyr12qevc00dzgkm9x94', u'2015-02-22 22:11:54')
[22/Feb/2015 22:11:54] INFO [app.views:33]  Configured project with domain Barcelona and keyword malaga
[22/Feb/2015 22:11:54] DEBUG [django.db.backends:91] (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
[22/Feb/2015 22:11:54] DEBUG [django.db.backends:91] (0.001) SELECT `app_domain`.`id`, `app_domain`.`name` FROM `app_domain` WHERE `app_domain`.`name` = 'Barcelona' LIMIT 21; args=(u'Barcelona',)
[22/Feb/2015 22:11:54] DEBUG [django.db.backends:91] (0.001) SELECT `app_topic`.`id`, `app_topic`.`name`, `app_topic`.`count`, `app_topic`.`domain_id` FROM `app_topic` WHERE `app_topic`.`name` = 'malaga' LIMIT 21; args=(u'malaga',)
[22/Feb/2015 22:11:54] INFO [app.domain.digger:24] Start streaming from key: malaga
[22/Feb/2015 22:11:57] INFO [app.domain.stream:24] New data arrived. Count is 0
[22/Feb/2015 22:11:57] INFO [app.domain.stream:30] {"created_at":"Mon Feb 23 01:11:57 +0000 2015","id":569665935537098752,"id_str":"569665935537098752","text":"Vem a\u00ed mais um epis\u00f3dio in\u00e9dito do #WWEnoFOXSports no #FOXSports2, \u00e0s 18h45. N\u00e3o perca!  http:\/\/t.co\/xkJv5QXLn6","source":"\u003ca href=\"http:\/\/www.slavemachine.com.br\" rel=\"nofollow\"\u003eSlave Machine Robot\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2931020086,"id_str":"2931020086","name":"Slave Machine","screen_name":"IAmSlaveMachine","location":"Palo Alto, CA","url":"https:\/\/twitter.com\/IAmSlaveMachine","description":"SlaveMachine","protected":false,"verified":false,"followers_count":1337,"friends_count":2014,"listed_count":7,"favourites_count":74024,"statuses_count":105714,"created_at":"Thu Dec 18 22:29:40 +0000 2014","utc_offset":-10800,"time_zone":"Buenos Aires","geo_enabled":true,"lang":"pt","contributors_enabled":false,"is_translator":false,"profile_background_color":"FA743E","profile_background_image_url":"http:\/\/pbs.twimg.com\/profile_background_images\/545711093016838145\/FjF9-lFt.jpeg","profile_background_image_url_https":"https:\/\/pbs.twimg.com\/profile_background_images\/545711093016838145\/FjF9-lFt.jpeg","profile_background_tile":false,"profile_link_color":"FA743E","profile_sidebar_border_color":"C0DEED","profile_sidebar_fill_color":"DDEEF6","profile_text_color":"333333","profile_use_background_image":true,"profile_image_url":"http:\/\/pbs.twimg.com\/profile_images\/546300487318777857\/u_rtW9Pt_normal.png","profile_image_url_https":"https:\/\/pbs.twimg.com\/profile_images\/546300487318777857\/u_rtW9Pt_normal.png","profile_banner_url":"https:\/\/pbs.twimg.com\/profile_banners\/2931020086\/1419082880","default_profile":false,"default_profile_image":false,"following":null,"follow_request_sent":null,"notifications":null},"geo":null,"coordinates":null,"place":null,"contributors":null,"retweet_count":0,"favorite_count":0,"entities":{"hashtags":[{"text":"WWEnoFOXSports","indices":[35,50]},{"text":"FOXSports2","indices":[54,65]}],"trends":[],"urls":[{"url":"http:\/\/t.co\/xkJv5QXLn6","expanded_url":"http:\/\/ay.gy\/138lH2","display_url":"ay.gy\/138lH2","indices":[89,111]}],"user_mentions":[],"symbols":[]},"favorited":false,"retweeted":false,"possibly_sensitive":false,"filter_level":"low","lang":"pt","timestamp_ms":"1424653917570"}

[22/Feb/2015 22:11:57] INFO [app.domain.stream:35] User has been save successfully
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) SELECT `app_user`.`id`, `app_user`.`user_id`, `app_user`.`name`, `app_user`.`screen_name`, `app_user`.`description`, `app_user`.`followers_count`, `app_user`.`friends_count`, `app_user`.`statuses_count`, `app_user`.`favourites_count`, `app_user`.`location`, `app_user`.`time_zone`, `app_user`.`created_at` FROM `app_user` WHERE `app_user`.`user_id` = 2931020086 LIMIT 21; args=(2931020086L,)
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) INSERT INTO `app_user` (`user_id`, `name`, `screen_name`, `description`, `followers_count`, `friends_count`, `statuses_count`, `favourites_count`, `location`, `time_zone`, `created_at`) VALUES (2931020086, 'Slave Machine', 'IAmSlaveMachine', 'SlaveMachine', 1337, 2014, 105714, 74024, 'Palo Alto, CA', 'Buenos Aires', '2014-12-18 22:29:40'); args=[2931020086L, 'Slave Machine', 'IAmSlaveMachine', 'SlaveMachine', 1337, 2014, 105714, 74024, 'Palo Alto, CA', 'Buenos Aires', u'2014-12-18 22:29:40']
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) SELECT `app_topic`.`id`, `app_topic`.`name`, `app_topic`.`count`, `app_topic`.`domain_id` FROM `app_topic` LIMIT 21; args=()
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.000) SELECT `app_tweet`.`id`, `app_tweet`.`tweetid`, `app_tweet`.`text`, `app_tweet`.`favorite_count`, `app_tweet`.`retweet_count`, `app_tweet`.`retweet_id`, `app_tweet`.`author_id`, `app_tweet`.`topic_id` FROM `app_tweet` WHERE `app_tweet`.`tweetid` = 569665935537098752 LIMIT 21; args=(569665935537098752L,)
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.015) INSERT INTO `app_tweet` (`tweetid`, `text`, `favorite_count`, `retweet_count`, `retweet_id`, `author_id`, `topic_id`) VALUES (569665935537098752, 'Vem a\\xed mais um epis\\xf3dio in\\xe9dito do #WWEnoFOXSports no #FOXSports2, \\xe0s 18h45. N\\xe3o perca!  http://t.co/xkJv5QXLn6', 0, 0, NULL, 6, 1); args=[569665935537098752L, 'Vem a\\xed mais um epis\\xf3dio in\\xe9dito do #WWEnoFOXSports no #FOXSports2, \\xe0s 18h45. N\\xe3o perca!  http://t.co/xkJv5QXLn6', 0, 0, None, 6, 1]
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) UPDATE `app_tweet` SET `tweetid` = 569665935537098752, `text` = 'Vem a\\xed mais um epis\\xf3dio in\\xe9dito do #WWEnoFOXSports no #FOXSports2, \\xe0s 18h45. N\\xe3o perca!  http://t.co/xkJv5QXLn6', `favorite_count` = 0, `retweet_count` = 0, `retweet_id` = NULL, `author_id` = 6, `topic_id` = 1 WHERE `app_tweet`.`id` = 3; args=(569665935537098752L, 'Vem a\\xed mais um epis\\xf3dio in\\xe9dito do #WWEnoFOXSports no #FOXSports2, \\xe0s 18h45. N\\xe3o perca!  http://t.co/xkJv5QXLn6', 0, 0, 6, 1, 3)
[22/Feb/2015 22:11:57] INFO [app.domain.stream:41] Tweet has been save successfully
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.002) SELECT `app_topic`.`id`, `app_topic`.`name`, `app_topic`.`count`, `app_topic`.`domain_id` FROM `app_topic` LIMIT 21; args=()
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.039) INSERT INTO `app_hashtag` (`name`, `count`, `topic_id`) VALUES ('WWEnoFOXSports', 0, 1); args=['WWEnoFOXSports', 0, 1]
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.012) SELECT `app_hashtag_tweets`.`tweet_id` FROM `app_hashtag_tweets` WHERE (`app_hashtag_tweets`.`tweet_id` IN (3) AND `app_hashtag_tweets`.`hashtag_id` = 1); args=(3, 1)
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.019) INSERT INTO `app_hashtag_tweets` (`hashtag_id`, `tweet_id`) VALUES (1, 3); args=(1, 3)
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) UPDATE `app_hashtag` SET `name` = 'WWEnoFOXSports', `count` = 0, `topic_id` = 1 WHERE `app_hashtag`.`id` = 1; args=('WWEnoFOXSports', 0, 1, 1)
[22/Feb/2015 22:11:57] INFO [app.domain.stream:46] Hashtag has been save successfully
[22/Feb/2015 22:11:57] INFO [app.domain.stream:48] --------------
[22/Feb/2015 22:11:57] INFO [app.domain.stream:24] New data arrived. Count is 1
[22/Feb/2015 22:11:57] INFO [app.domain.stream:30] {"created_at":"Mon Feb 23 01:11:57 +0000 2015","id":569665934970851328,"id_str":"569665934970851328","text":"Cry conjunction euphony out the eyes relative to straight a extant goodwife: yktm","source":"\u003ca href=\"http:\/\/twitterfeed.com\" rel=\"nofollow\"\u003etwitterfeed\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1240727480,"id_str":"1240727480","name":"EricksonLuke","screen_name":"EricksonLuke2","location":"","url":null,"description":null,"protected":false,"verified":false,"followers_count":92,"friends_count":1,"listed_count":0,"favourites_count":0,"statuses_count":73353,"created_at":"Mon Mar 04 07:04:05 +0000 2013","utc_offset":7200,"time_zone":"Istanbul","geo_enabled":false,"lang":"pt","contributors_enabled":false,"is_translator":false,"profile_background_color":"C0DEED","profile_background_image_url":"http:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png","profile_background_image_url_https":"https:\/\/abs.twimg.com\/images\/themes\/theme1\/bg.png","profile_background_tile":false,"profile_link_color":"0084B4","profile_sidebar_border_color":"C0DEED","profile_sidebar_fill_color":"DDEEF6","profile_text_color":"333333","profile_use_background_image":true,"profile_image_url":"http:\/\/pbs.twimg.com\/profile_images\/3501202501\/11230f99529b4bbdf606d29bc581d130_normal.jpeg","profile_image_url_https":"https:\/\/pbs.twimg.com\/profile_images\/3501202501\/11230f99529b4bbdf606d29bc581d130_normal.jpeg","default_profile":true,"default_profile_image":false,"following":null,"follow_request_sent":null,"notifications":null},"geo":null,"coordinates":null,"place":null,"contributors":null,"retweet_count":0,"favorite_count":0,"entities":{"hashtags":[],"trends":[],"urls":[],"user_mentions":[],"symbols":[]},"favorited":false,"retweeted":false,"possibly_sensitive":false,"filter_level":"low","lang":"en","timestamp_ms":"1424653917435"}

[22/Feb/2015 22:11:57] INFO [app.domain.stream:35] User has been save successfully
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.020) SELECT `app_user`.`id`, `app_user`.`user_id`, `app_user`.`name`, `app_user`.`screen_name`, `app_user`.`description`, `app_user`.`followers_count`, `app_user`.`friends_count`, `app_user`.`statuses_count`, `app_user`.`favourites_count`, `app_user`.`location`, `app_user`.`time_zone`, `app_user`.`created_at` FROM `app_user` WHERE `app_user`.`user_id` = 1240727480 LIMIT 21; args=(1240727480,)
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) INSERT INTO `app_user` (`user_id`, `name`, `screen_name`, `description`, `followers_count`, `friends_count`, `statuses_count`, `favourites_count`, `location`, `time_zone`, `created_at`) VALUES (1240727480, 'EricksonLuke', 'EricksonLuke2', NULL, 92, 1, 73353, 0, '', 'Istanbul', '2013-03-04 07:04:05'); args=[1240727480, 'EricksonLuke', 'EricksonLuke2', None, 92, 1, 73353, 0, '', 'Istanbul', u'2013-03-04 07:04:05']
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.000) SELECT `app_topic`.`id`, `app_topic`.`name`, `app_topic`.`count`, `app_topic`.`domain_id` FROM `app_topic` LIMIT 21; args=()
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.000) SELECT `app_tweet`.`id`, `app_tweet`.`tweetid`, `app_tweet`.`text`, `app_tweet`.`favorite_count`, `app_tweet`.`retweet_count`, `app_tweet`.`retweet_id`, `app_tweet`.`author_id`, `app_tweet`.`topic_id` FROM `app_tweet` WHERE `app_tweet`.`tweetid` = 569665934970851328 LIMIT 21; args=(569665934970851328L,)
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.000) INSERT INTO `app_tweet` (`tweetid`, `text`, `favorite_count`, `retweet_count`, `retweet_id`, `author_id`, `topic_id`) VALUES (569665934970851328, 'Cry conjunction euphony out the eyes relative to straight a extant goodwife: yktm', 0, 0, NULL, 7, 1); args=[569665934970851328L, 'Cry conjunction euphony out the eyes relative to straight a extant goodwife: yktm', 0, 0, None, 7, 1]
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.011) UPDATE `app_tweet` SET `tweetid` = 569665934970851328, `text` = 'Cry conjunction euphony out the eyes relative to straight a extant goodwife: yktm', `favorite_count` = 0, `retweet_count` = 0, `retweet_id` = NULL, `author_id` = 7, `topic_id` = 1 WHERE `app_tweet`.`id` = 4; args=(569665934970851328L, 'Cry conjunction euphony out the eyes relative to straight a extant goodwife: yktm', 0, 0, 7, 1, 4)
[22/Feb/2015 22:11:57] INFO [app.domain.stream:41] Tweet has been save successfully
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) SELECT `app_topic`.`id`, `app_topic`.`name`, `app_topic`.`count`, `app_topic`.`domain_id` FROM `app_topic` LIMIT 21; args=()
[22/Feb/2015 22:11:57] ERROR [app.domain.digger:28] list index out of range
[22/Feb/2015 22:11:57] INFO [app.domain.digger:29] Finish Digger from key: malaga
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) SELECT `celery_taskmeta`.`id`, `celery_taskmeta`.`task_id`, `celery_taskmeta`.`status`, `celery_taskmeta`.`result`, `celery_taskmeta`.`date_done`, `celery_taskmeta`.`traceback`, `celery_taskmeta`.`hidden`, `celery_taskmeta`.`meta` FROM `celery_taskmeta` WHERE `celery_taskmeta`.`task_id` = '888f5a2c-3d40-4b0d-9a16-834267bae898' LIMIT 21; args=('888f5a2c-3d40-4b0d-9a16-834267bae898',)
[22/Feb/2015 22:11:57] DEBUG [django.db.backends:91] (0.001) INSERT INTO `celery_taskmeta` (`task_id`, `status`, `result`, `date_done`, `traceback`, `hidden`, `meta`) VALUES ('888f5a2c-3d40-4b0d-9a16-834267bae898', 'FAILURE', 'gAJjZXhjZXB0aW9ucwpUeXBlRXJyb3IKcQFVMnN0YXJ0X2RpZ2dlcigpIHRha2VzIGV4YWN0bHkgMyBhcmd1bWVudHMgKDQgZ2l2ZW4pcQKFcQNScQQu', '2015-02-22 22:11:57', 'Traceback (most recent call last):\n  File \"C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py\", line 240, in trace_task\n    R = retval = fun(*args, **kwargs)\n  File \"C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py\", line 438, in __protected_call__\n    return self.run(*args, **kwargs)\n  File \"C:\\Users\\Marina\\Desktop\\TIP\\tip.twitter\\twitter\\twitter\\app\\domain\\tasks.py\", line 45, in start_digger\n    start_digger(self, dominio, topic)\n  File \"C:\\Python27\\lib\\site-packages\\celery\\local.py\", line 167, in <lambda>\n    __call__ = lambda x, *a, **kw: x._get_current_object()(*a, **kw)\n  File \"C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py\", line 439, in __protected_call__\n    return orig(self, *args, **kwargs)\n  File \"C:\\Python27\\lib\\site-packages\\celery\\app\\task.py\", line 420, in __call__\n    return self.run(*args, **kwargs)\nTypeError: start_digger() takes exactly 3 arguments (4 given)\n', 0, 'eJxrYKotZIzgYGBgSM7IzEkpSs0rZIotZC7WAwBWuwcA'); args=['888f5a2c-3d40-4b0d-9a16-834267bae898', 'FAILURE', u'gAJjZXhjZXB0aW9ucwpUeXBlRXJyb3IKcQFVMnN0YXJ0X2RpZ2dlcigpIHRha2VzIGV4YWN0bHkgMyBhcmd1bWVudHMgKDQgZ2l2ZW4pcQKFcQNScQQu', u'2015-02-22 22:11:57', 'Traceback (most recent call last):\n  File "C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py", line 240, in trace_task\n    R = retval = fun(*args, **kwargs)\n  File "C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py", line 438, in __protected_call__\n    return self.run(*args, **kwargs)\n  File "C:\\Users\\Marina\\Desktop\\TIP\\tip.twitter\\twitter\\twitter\\app\\domain\\tasks.py", line 45, in start_digger\n    start_digger(self, dominio, topic)\n  File "C:\\Python27\\lib\\site-packages\\celery\\local.py", line 167, in <lambda>\n    __call__ = lambda x, *a, **kw: x._get_current_object()(*a, **kw)\n  File "C:\\Python27\\lib\\site-packages\\celery\\app\\trace.py", line 439, in __protected_call__\n    return orig(self, *args, **kwargs)\n  File "C:\\Python27\\lib\\site-packages\\celery\\app\\task.py", line 420, in __call__\n    return self.run(*args, **kwargs)\nTypeError: start_digger() takes exactly 3 arguments (4 given)\n', False, u'eJxrYKotZIzgYGBgSM7IzEkpSs0rZIotZC7WAwBWuwcA']
[22/Feb/2015 22:13:16] DEBUG [django.db.backends:91] (0.001) SET SQL_AUTO_IS_NULL = 0; args=None
[22/Feb/2015 22:13:16] DEBUG [django.db.backends:91] (0.036) SHOW TABLES; args=None
[22/Feb/2015 22:13:16] DEBUG [django.db.backends:91] (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
